src/ert/scheduler/job.py

```diff
@@ -47,12 +47,18 @@ class Job:
     (LSF, PBS, SLURM, etc.)
     """
 
-    def __init__(self, scheduler: Scheduler, real: Realization) -> None:
+    def __init__(
+        self,
+        scheduler: Scheduler,
+        real: Realization,
+        callback_timeout: Optional[Callable[[int], None]] = None,
+    ) -> None:
         self.real = real
         self.started = asyncio.Event()
         self.returncode: asyncio.Future[int] = asyncio.Future()
         self.aborted = asyncio.Event()
         self._scheduler = scheduler
+        self._callback_timeout: Optional[Callable[[int], None]] = callback_timeout
```

Comment:
Do we need this function as an attribute?

Reply:
The timeout is set up by `_legacy.py` and we need to keep in sync with how the existing job_queue operates.

Reply:
Maybe we can have a
```py
if isinstance(self._job_queue, Scheduler):
```
in `_legacy.py`? I reckon that is less dirty than having this `callback_timeout`.

Reply:
Fixed!